analytics/location_heatmaps/geo_utils.py

```diff
@@ -278,141 +296,124 @@ def split_regions(tree_prefix_list,
   Args:
       tree_prefix_list: matches vector id to the tree prefix.
       vector_counts: vector values aggregated from the users.
-      threshold: threshold value used to split the nodes.
+      split_threshold: threshold value used to split the nodes.
       image_bit_level: stopping criteria once the final resolution is reached.
       collapse_threshold: threshold value used to collapse the nodes.
+      expand_all: expand all regions,
```

Comment:
can we achieve this functionality by just passing split_threshold = -np.inf and eliminate the extra parameter & special-casing?

Reply:
yeah, great idea, fixed